networking.md

```diff
@@ -4,17 +4,17 @@ title: Making HTTP Networking Requests in Flutter
 permalink: /networking/
 ---
 
-This page discusses how to make HTTP networking requests in Flutter. For
+This page describes how to make HTTP networking requests in Flutter. For
 HTTP servers and sockets, see [dart:io][dartio].
```

Comment:
I don't think we'd promote that you can build HTTP servers in Flutter :) so it's OK to scope this to just "For more on how to use TCP and UDP sockets from within Flutter, ..."